lib/Controller/SiteController.php

```diff
@@ -61,10 +61,10 @@ public function __construct(string $appName,
 	 *
 	 * @return TemplateResponse|RedirectResponse
 	 */
-	public function showPage(int $id) {
+	public function showPage(int $id, ?string $path = null) {
```

Comment:
```suggestion
	public function showPage(int $id, string $path) {
```
With the requirement in the route you don't need to have it nullable